src/hooks/usePipelineRunsForApplication.ts

```diff
@@ -15,23 +16,31 @@ export const useLatestPipelineRunForComponent = (
     groupVersionKind: PipelineRunGroupVersionKind,
     namespace,
     isList: true,
+    selector: {
+      matchExpressions: [
+        {
+          key: PipelineRunLabel.COMPONENT,
+          operator: 'Equals',
+          values: [component.metadata.name],
+        },
+        {
+          key: PipelineRunLabel.APPLICATION,
+          operator: 'Equals',
+          values: [component.spec.application],
+        },
```

Comment:
This can be simplified by using `matchLabels`. `matchExpressions` is good when you need an operator other than `Equals`.